src/mappings/events/vaults.ts

```diff
@@ -89,8 +93,26 @@ export const vaultsEventKit = (block: any, data: any, module: string, path: stri
       );
     }
 
-    vault.coin = payload?.locked?.__brand;
-    vault.denom = payload?.locked?.__brand;
+    const vaultGovernanceId = id.split('.').slice(0, 4).join('.') + '.governance';
+    const vaultManagerGovernance = await VaultManagerGovernance.get(vaultGovernanceId);
```

Comment:
I think we should add optional chaining on line 96:
```
const vaultGovernanceId = id?.split('.')?.slice(0, 4).join('.') + '.governance';
```

Reply:
id here should exist in all cases. if we put optional chaining here, we will get an invalid id. crashing in case the id format is invalid would be the preferred option here

Reply:
this function is only triggered from line: src/mappings/mappingHandlers.ts:214 where we perform a regex check on the path. so there is no way it is falsy.
in any case i've added a regex check just to be safe